--- gqradio-1.9.1/gqradio.desktop~	Wed Jul 28 01:14:15 2004
+++ gqradio-1.9.1/gqradio.desktop	Wed Jul 28 01:14:15 2004
@@ -2,13 +2,15 @@
 Name=GQradio
 Comment=Radio tuner
 Comment[fr]=Tuner radio
+Comment[pl]=Tuner radiowy
 Exec=gqradio
 Icon=gqradio.png
 Info=Radio tuner app.
 Info[fr]=Application tuner radio
-Terminal=0
+Terminal=false
 Type=Application
 StartupNotify=false
 StartupWMClass=gqradio
 Encoding=UTF-8
-Categories=Application;AudioVideo;
+Categories=GTK;Audio;
+# vi: encoding=utf-8
